### ISSUE TITLE
Does not work in Statamic 3.4.5

### ISSUE THREAD

**Issue by SG-fabian** [MarkKravchuk/statamic-activity-book#1]
Hi,

I've tried to use this plugin but unfortunately it does not work anymore in Statamic 3.4.5.

**Comment by lauritsen** [comment]
Hi @MarkKravchuk 
I have the same issue. It seems to be related to how the utilities are registered, which was changed from v3.3 to v3.4: https://statamic.dev/upgrade-guide/3-3-to-3-4#utility-fluent-methods

I think if you make the following change in ServiceProvider.php it should work. 
Don't know how or if you wanna manage different version (I am new to statamic...), but would also be happy to do a PR with this change if you want that.
```
// Create UI page for addon, under Utilities tab
Utility::extend(function () {
    Utility::register('activity-book')
        ->title(__('ActivityBook'))
        ->icon('book-pages')
        ->description(__('View all activities made to the content of the website'))
        ->routes(function (Router $router) {
            // Only 1 page to be displayed
            $router->get('/', [ActivityBookViewController::class, 'show'])->name('show');
        });
});
```

**Comment by MarkKravchuk** [comment]
@lauritsen Hi!

This package was created as a small university project. I am personally not using it, however if you find it useful in any way for your projects, feel free to create PR's which I will merge

**Comment by MarkKravchuk** [comment]
This issue has been fix by @ivanpisa and released in version 1.0.1
